### DISCUSSION TITLE
[@types/culori] Missing color space: Xyb

### DISCUSSION THREAD

**Discussion by Jeremy-Knudsen** [DefinitelyTyped/DefinitelyTyped#64818] [Issues with a @types package]
Culori JS API was updated with a new color space "Xyb" along with other features. The types package does not include the Xyb color space: https://culorijs.org/api/#color-spaces

[a comment by an automated account is not shown]

**Reply by bijela-gora** [reply]
Hi @Jeremy-Knudsen 

A pull request are more then welcome!

**Comment by bijela-gora** [comment]
Hi @Jeremy-Knudsen

A pull request are more then welcome!
